core/src/messages.cpp

```diff
@@ -450,6 +446,34 @@ void group_by_fragment_inplace(curtain_query& query) {
         std::tie(dim0s[i], dim1s[i]) = pairs[i];
 }
 
+std::pair< int, int > utm_to_lineno(
+        int trace0_iline,
+        int trace0_xline,
+        float trace0_x,
+        float trace0_y,
+        float xlinestep_x,
+        float xlinestep_y,
+        float ilinestep_x,
+        float ilinestep_y,
+        float x,
+        float y
+) noexcept (false) {
+    const double det =
+            1 / (xlinestep_x * ilinestep_y - ilinestep_x * xlinestep_y);
```

Comment:
Hmm, I think I made the geo info optional? So it should not be set if it is not present? Where would this be set to 0?

Reply:
You made it optional for old manifests (which btw means new client/old data should be a version test).
But as your scan results show https://github.com/equinor/oneseismic/pull/526/commits/6cc7615501eb94d1b87a694078112ff53ac6f6eb the geo data will be present, it will just be zeroes. I assumed this is a correct result and should be trusted, probably if no cdp data is available? This causes division-by-zero thing.